src/main/java/org/openhab/tools/analysis/checkstyle/DeclarativeServicesDependencyInjectionCheck.java

```diff
@@ -44,6 +44,11 @@
     }
 
     @Override
+    public int[] getAcceptableTokens() {
+        return getDefaultTokens();
```

Comment:
As this check is too specific, I don't think that there are any cases when the acceptable tokens could be different that the default ones.